Makefile

```
@@ -44,33 +49,60 @@ test:
 functional-test:
 	go test -mod=vendor -timeout=120s -v -tags functional -cover ./local-container-endpoints/handlers/functional_tests/...
 
-$(LINUX_BINARY): $(SOURCES)
+$(AMD_BINARY): $(SOURCES)
 	@mkdir -p ./bin/linux-amd64
 	TARGET_GOOS=linux GOARCH=amd64 ./scripts/build_binary.sh ./bin/linux-amd64
-	@echo "Built local-container-endpoints for linux"
+	@echo "Built local-container-endpoints for linux-amd64"
+
+$(AMD_BINARY): $(SOURCES)
+	@mkdir -p ./bin/linux-arm64
+	TARGET_GOOS=linux GOARCH=arm64 ./scripts/build_binary.sh ./bin/linux-arm64
+	@echo "Built local-container-endpoints for linux-arm64"
 
 .PHONY: release
-release:
+release: release-amd release-arm
+
+.PHONY: release-amd
+release-amd:
+	docker run -v $(shell pwd):/usr/src/app/src/github.com/awslabs/amazon-ecs-local-container-endpoints \
+		--workdir=/usr/src/app/src/github.com/awslabs/amazon-ecs-local-container-endpoints \
+		--env GOPATH=/usr/src/app \
+		--env ECS_RELEASE=cleanbuild \
+		golang:$(GO_VERSION) make $(AMD_BINARY)
+	docker build -t amazon/amazon-ecs-local-container-endpoints:latest-amd64 .
+	docker tag amazon/amazon-ecs-local-container-endpoints:latest amazon/amazon-ecs-local-container-endpoints:$(TAG)-amd64
+	docker tag amazon/amazon-ecs-local-container-endpoints:latest amazon/amazon-ecs-local-container-endpoints:$(VERSION)-amd64
```

does this need to be like this?
```suggestion
	docker tag amazon/amazon-ecs-local-container-endpoints:latest-amd64 amazon/amazon-ecs-local-container-endpoints:$(TAG)-amd64
	docker tag amazon/amazon-ecs-local-container-endpoints:latest-amd64 amazon/amazon-ecs-local-container-endpoints:$(VERSION)-amd64
```

---

ahhh good catch! fixed it both here and for the arm64 images.